.github/workflows/release-branch.yml

```diff
@@ -0,0 +1,55 @@
+# Create release branch
+# TODO after GitHub add permission for action-bot to commit to the protected branches - please merge release-* workflow into one
+
+name: Create release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Release version:"
+        required: true
+
+jobs:
+  create-release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.inputs.version }}
+      TAG: v${{ github.event.inputs.version }}
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 #required for tags
+
+    - name: Create all-in-one configuration
+      uses: ./.github/actions/all-in-one
+      with:
+        IMAGE_URL: ${{ secrets.DOCKER_REGISTRY }}/${{ secrets.DOCKER_REPO }}:${{ env.VERSION }}
+        version: ${{ env.VERSION }}
+
+    - name: Create branch and push it
+      run: |
+        git checkout -b "release/${VERSION}"
+        git push origin "release/${VERSION}"
+
+    - name: Commit file'n'sign
+      uses: ./.github/actions/put-contents
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        FILE_TO_COMMIT: "deploy/all-in-one.yaml"
```

Comment:
btw the rename I've suggested above refers only to dev scripts - this file name should be clear to the public I believe 👍